Water Drink Water/src/clients/blazor.wa.tbd/Services/AuthService.cs

```diff
@@ -62,4 +62,11 @@ public async Task<bool> Authenticate(string email, string password)
 
         return true;
     }
+
+    public async Task<bool> Register(string name, string email, string password)
+    {
+        var response = await client.PostAsJsonAsync("api/registration", new { name, email, password });
+
+        return response.IsSuccessStatusCode;
+    }
 }
```

Comment:
Final note and I understand that is normally not really put in here. This is the first time I did a review, hope it helped a little.

Reply:
Absolutely, thank you for taking the time, this was very much appreciated and gave us some discussion points

Reply:
Hello, fly on the wall here. This is a great review.